packages/medusa/src/api/routes/admin/batch/index.ts

```diff
@@ -1,18 +1,35 @@
 import { Router } from "express"
 import { BatchJob } from "../../../.."
 import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
-import middlewares from "../../../middlewares"
+import middlewares, {
+  getRequestedBatchJob,
+  canAccessBatchJob,
+} from "../../../middlewares"
 
 export default (app) => {
   const route = Router()
 
-  app.use("/batch", route)
+  app.use("/batch-jobs", route)
 
   route.get(
     "/",
     middlewares.normalizeQuery(),
     middlewares.wrap(require("./list-batch-jobs").default)
   )
+  route.post("/", middlewares.wrap(require("./create-batch-job").default))
+
+  const routerOnBatch = Router()
+  route.use("/:id", getRequestedBatchJob, canAccessBatchJob, routerOnBatch)
+  routerOnBatch.get("/", middlewares.wrap(require("./get-batch-job").default))
+  routerOnBatch.post(
+    "/confirm-processing",
```

Comment:
Confirm or process? Since it emit an event for processing?

Reply:
I think it will depend on the comment I made below about adding an additional status:

- if we have two statuses, i.e. "confirmed" -> "processing" then I think it should be `/confirm`
- if we go directly to "processing" as a result of the API call I think `/process` is good.

I am leaning towards introducing the additional status though as I think it gives a more intuitive flow for the API consumer. 

The mental flow that I am thinking of from the API consumer's point of view is: 
"first I create, then I get a picture of what will happen, and then I confirm and I can get my results"

The other story is a bit harder to follow:
"first I create, then I get a picture of what will happen, and then I process and I can get my results"

I think the thing might be that "confirm" is the action that the user takes and "process" is the action that the worker starts so it seems more intuitive to name it after the user action.

Reply:
Yes I understand, just to add an info, the idea was to get the same flow no matter if a job is dry_run or not and use the dry_run as our segregated value